R/mod_gene_expression_heatmap.R

```diff
@@ -194,7 +194,22 @@ mod_gene_expression_heatmap_server <- function(input, output, session, gene_expr
   # Save output ----
 
   gene_expression_data_download <- shiny::reactive({
-    filtered_gene_expressions()
+    # Select and rename columns
+    data_cols <- filtered_gene_expressions() %>%
+      dplyr::select(
+        `Mouse Model` = .data$mouse_model,
+        Tissue = .data$tissue,
+        Sex = .data$sex,
+        Age = .data$age,
+        Gene = .data$gene,
+        `Differentially Expressed` = .data$diff_expressed,
+        `Log2 Fold Change` = .data$log2foldchange,
+        `Adjusted P-Value` = .data$padj
```

Comment:
diff_expressed in snake case, but the other variables in lower case?  Not a request for change, but in data-driven applications I like using snake_case for variables (it's just easier to read).

Reply:
That's also debatable.  I should say that I find it easier to read! 🤫

Reply:
I prefer snake_case too! I'm not sure why I kept log2foldchange as is - that might be the name in the actual data and I didn't want to change it since it was already so close... but I'll take a pass over the data sources and app, generally I prefer snake_case >>>>>> any other case!